_rules/html-page-title-2779a5.md

```diff
@@ -56,6 +56,7 @@ _There are no major accessibility support issues known for this rule._
 - [Understanding Success Criterion 2.4.2: Page Titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled)
 - [G88: Providing descriptive titles for Web pages](https://www.w3.org/WAI/WCAG21/Techniques/general/G88)
 - [H25: Providing a title using the title element](https://www.w3.org/WAI/WCAG21/Techniques/html/H25)
+- [HTML 5.2: the `title` element](https://www.w3.org/TR/html52/document-metadata.html#the-title-element)
```

Comment:
Other concepts in this rule are linked to WHATWG spec (descendant, document). Should this be WHATWG instead?
(we haven't really solved #879).

Reply:
I use the same link to the title element that is in several other places in this document. Yes we should shift over to WHATWG spec, but that's not what this PR is for.

Reply:
OK.